docs/README.md

```diff
@@ -66,11 +73,15 @@ $ sudo pip install 'sphinx<3.1.0' mkdocs numpy pydata_sphinx_theme ipython nbsph
 
 ### R API Documentation (Optional)
 
-If you'd like to generate R API documentation, you'll need to [install Pandoc](https://pandoc.org/installing.html)
-and install these libraries:
+If you'd like to generate R API documentation, you'll need to install these packages and libraries:
+
+```sh
+$ sudo apt install libssl-dev libcurl4-openssl-dev pandoc libfontconfig1-dev libharfbuzz-dev \
```

Comment:
Hm, I think we should better make it independent from the OS

Reply:
Do you have any suggestion? I can only suggest adding a Dockerfile similar to [this one](https://github.com/khalidmammadov/spark/blob/e9cec4091b159e1c0c6c44a1fb816ca16a77e9f5/docs/Dockerfile) to build and test the changes or omit these installs as they are for linux? In the last case it makes again not complete and one needs to figure it out what to install every time.